.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
```

Comment:
Can we matrix this on a few versions of Ruby while we're at it? Similar to what I do with rice. https://github.com/ruby-rice/rice/blob/master/.github/workflows/testing.yml#L15

Reply:
https://github.com/sebyx07/rbgccxml/actions/runs/10754739740/job/29825700177

i also added 2.7 - to check, but IMO you should support 3.0.0(2020)

Reply:
https://github.com/sebyx07/rbgccxml/actions/runs/10754760844 - to run 3.0+